test/integration/gateways/securion_pay_test.exs

```diff
@@ -0,0 +1,78 @@
+defmodule Gringotts.Integration.Gateways.SecurionPayTest do
+  use ExUnit.Case, async: true
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
+
+  alias Gringotts.{
+    CreditCard
+  }
+
+  alias Gringotts.Gateways.SecurionPay, as: Gateway
+
+  @moduletag integration: false
+
+  @amount Money.new(42, :EUR)
+
+  @bad_card %CreditCard{
+    first_name: "Harry",
+    last_name: "Potter",
+    number: "4200000000000000",
+    year: 2009,
+    month: 12,
+    verification_code: "123",
+    brand: "VISA"
+  }
+
+  @good_card %CreditCard{
+    first_name: "Harry",
+    last_name: "Potter",
+    number: "4200000000000000",
+    year: 2019,
+    month: 12,
+    verification_code: "123",
+    brand: "VISA"
+  }
+
+  @opts [
+    config: [secret_key: "sk_test_Ff2Jpq59sSWT7qmI9atii5zR"],
+    customer_id: "cust_zpYEBK396q3rvIBZYc3PIDwT"
+  ]
+
+  @bad_opts [config: [secret_key: "sk_test_Ff2Jpq59sSWT7qmI9atii5zR"]]
+
+  @card_id "card_NH4bn2T2h2QyXvW1fsRkZo8O"
+
+  test "[authorize] with CreditCard" do
```

Comment:
Please put these tests in a describe block, perhaps with the name `[authorize]`.